lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt

```diff
@@ -82,34 +83,32 @@ netifaces==0.10.9
 networkx==1.11
 nodeenv==1.3.3
 nose==1.3.7
-numpy==1.16.1
+numpy==1.16.2
 oauthlib==3.0.1
 openstacksdk==0.17.0
 os-client-config==1.31.2
-os-service-types==1.5.0
-osc-lib==1.12.0
-oslo.config==6.8.0
-oslo.context==2.22.0
-oslo.i18n==3.23.0
-oslo.log==3.42.2
-oslo.serialization==2.28.1
-oslo.utils==3.40.2
+os-service-types==1.6.0
+osc-lib==1.12.1
+oslo.config==6.8.1
+oslo.context==2.22.1
+oslo.i18n==3.23.1
+oslo.log==3.42.3
+oslo.serialization==2.28.2
+oslo.utils==3.40.3
 packaging==19.0
 paramiko==2.4.2
 parsley==1.3
-paste==3.0.6
+paste==3.0.7
 pastedeploy==2.0.1
-pastescript==3.0.0
-pbr==5.1.2
+pastescript==3.1.0
+pbr==5.1.3
 prettytable==0.7.2
-psutil==5.5.1
+psutil==5.6.0
```

Comment:
This one needs new wheels: https://github.com/galaxyproject/starforge-recipes/pull/40